modules/openxr/doc_classes/OpenXRBindingModifier.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="OpenXRBindingModifier" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		Binding modifier base class.
+	</brief_description>
+	<description>
+		Binding modifier base class. Subclasses implement various modifiers that alter how an OpenXR runtime processes inputs.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="_get_description" qualifiers="virtual const">
+			<return type="String" />
+			<description>
+				Return the description of this class that is used for the title bar of the binding modifier editor.
+			</description>
+		</method>
+		<method name="_get_ip_modification" qualifiers="virtual">
+			<return type="PackedByteArray" />
+			<description>
+				Returns the data that is sent to OpenXR when submitting the suggested interacting bindings this modifier is a part of.
+				[b]Note:[/b] This must be data compatible with a [code]XrBindingModificationBaseHeaderKHR[/code] structure.
+			</description>
+		</method>
+		<method name="_record_on_binding" qualifiers="virtual const">
+			<return type="bool" />
+			<description>
+				Return [code]false[/code] if binding modifiers of this type are recorded on an interaction profile.
+				Return [code]true[/code] if binding modifiers of this type are recorded on bindings within an interaction profile.
```

Comment:
The description is a bit confusing.

Reply:
Basically there are two types of binding modifiers, those recorded on the interaction profile, and those recorded against a specific binding on the interaction profile. This flag simply indicates which type we're dealing with.
This really won't get a lot cleared until more binding modifiers become public.